### PR TITLE
feat: add filters support for custom feeds

### PR DESCRIPTION
**Pull request by capJavert**
AS-323

- add `feedId` to both filters mutations so filters can be added to custom feeds
- `feedId` is optional to preserve BC with my feed

### CODE REVIEW

File: src/schema/feeds.ts [hunk 1]
```diff
@@ -1021,14 +1041,19 @@ const applyFeedPagingWithPin = (
...
+}: {
+  ctx: Context;
+  info: GraphQLResolveInfo;
+  feedId: string;
+}): Promise<GQLFeedSettings> => {
   const res = await graphorm.query<GQLFeedSettings>(ctx, info, (builder) => {
     builder.queryBuilder = builder.queryBuilder.andWhere(
-      `"${builder.alias}".id = :userId AND "${builder.alias}"."userId" = :userId`,
-      { userId: ctx.userId },
+      `"${builder.alias}".id = :feedId AND "${builder.alias}"."userId" = :userId`,
```
Comment (rebelchris) [inline src/schema/feeds.ts:1055]: Don't you need to do the `feedId || userId` here as well?

**Review by rebelchris — APPROVED** [review]
Non blocking comment, just making sure :)

File: src/schema/feeds.ts (hunk 1 above)
Comment (capJavert) [reply src/schema/feeds.ts:1055]: Yeah, but I did it by modfying the `getFeedSettings` to accept `feedId` as well so it is passed correctly from the outside.

**Review by nensidosari — APPROVED** [review]
Looks good

**Review by omBratteng — APPROVED** [review]

**Pull request merged**